crates/sui-core/src/authority.rs

```diff
@@ -661,15 +664,7 @@ impl AuthorityState {
         // to do this, since the false contention can be made arbitrarily low (no cost for 1.0 -
         // epsilon of txes) while solutions without false contention have slightly higher cost
         // for every tx.
-        let span = tracing::debug_span!(
```

Comment:
This debug span is not effective since there is no logging within acquire_tx_guard